admin_guide/persistent_storage_nfs.adoc

```diff
@@ -6,52 +6,34 @@
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 == Overview
-This topic describes how to provision an OpenShift cluster with persistent storage using NFS.  Some familiarity with Kubernetes and NFS is assumed.
+You can provision your OpenShift cluster with persistent storage using NFS. Some
+familiarity with Kubernetes and NFS is assumed.
 
-The Kubernetes
-link:../dev_guide/volumes.html[Persistent Volume] framework allows
-administrators to provision a cluster with persistent storage and gives users a way to request those resources without having
-any knowledge of the underlying infrastructure.
-
-IMPORTANT: High-availability of a storage in the infrastructure is left to the underlying storage provider.
+The Kubernetes link:../dev_guide/volumes.html[Persistent Volume] framework
+allows administrators to provision a cluster with persistent storage and gives
+users a way to request those resources without having any knowledge of the
+underlying infrastructure.
```

Comment:
I didn't think that term through too much, but it refers to users from an OpenShift object perspective, not sure how closely we want it to align that to actual people. "Regular users" as in the object can be given cluster admin privileges so it's not really the distinction we're looking for here.

Really the hardest distinction to draw is probably between the actual person and their OpenShift persona/user. Maybe "person" and "account" is the right distinction. Probably not something we can hash out today :)